src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -193,7 +197,8 @@ export class PlagiarismCaseInstructorDetailViewComponent implements OnInit, OnDe
         this.createdPost.content = this.translateService.instant('artemisApp.plagiarism.plagiarismCases.notification.body', {
             student: studentName,
             exercise: exerciseTitle,
-            course: courseTitle,
+            inCourseOrExam: 'artemisApp.plagiarism.plagiarismCases.notification.' + (belongsToExam ? 'inExam' : 'inCourse'),
```

Comment:
I think you're missing a call to the translation service here.

Reply:
Thanks for noticing! I fixed it at https://github.com/ls1intum/Artemis/pull/5620/commits/3671d7574d8d9c3a514af403d21939aaea4a6712